targets/TARGET_STM/README.md

```diff
@@ -377,6 +409,33 @@ You can change this in you local mbed_app.json:
 ```
 
 
+### Sleep feature
+
+ST MCUs feature several low-power modes, please check Reference Manual of each one for more details.
+
+- MBED sleep mode is usually mapped on ST SLEEP mode:
+  - CPU clock is off
+  - all peripherals can run and wake up the CPU when an interrupt or an event
+occurs
+
+- MBED deepsleep mode is mapped on ST STOP2 mode:
+  - all clocks in the VCORE domain are stopped
+  - the PLL, the MSI, the HSI and the HSE are disabled
+  - the LSI and the LSE can be kept running
+  - RTC can remain active
+
+Reminder related mbed-os: https://os.mbed.com/docs/mbed-os/latest/apis/power-management-sleep.html
+- debug profile is disabling deepsleep
+- deepsleep can also be disabled by application or drivers using sleep_manager_lock_deep_sleep()
+
+To enable the tracing, it is recommended to **not** define MBED_SLEEP_TRACING_ENABLED macro,
+but **MBED_SLEEP_STAT_ENABLED** macro.
```

Comment:
To my question above, why is not recommended/ Shoudn't the one be fixed rather than create two (it might confuse people) ?

Reply:
Add here some details why stats only.